R/pkg/R/DataFrame.R

```diff
@@ -510,9 +510,7 @@ setMethod("registerTempTable",
 #'
 #' Insert the contents of a SparkDataFrame into a table registered in the current SparkSession.
 #'
-#' @param x A SparkDataFrame
-#' @param tableName A character vector containing the name of the table
```

Comment:
Yes, I agree that ideally the doc should be kept near the function definition/body, and that's consistent with many other functions. So then the only issue is `...` would become first in the argument list of the doc. I'm not sure if that would bother the user to some extent.